schema/GeneticVariant_Alt_ID_Database_properties.mcf

```diff
@@ -0,0 +1,1401 @@
+
+Node: dcid:leiden_Muscular_Dystrophy_(MYL2)_ID
+name: "leiden_Muscular_Dystrophy_(MYL2)_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database leiden_Muscular_Dystrophy_(MYL2) for a genetic variant."
+
+Node: dcid:aRUP_Laboratories_Molecular_Genetics_and_Genomics_ARUP_Laboratories_ID
+name: "aRUP_Laboratories_Molecular_Genetics_and_Genomics_ARUP_Laboratories_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database aRUP_Laboratories_Molecular_Genetics_and_Genomics_ARUP_Laboratories for a genetic variant."
+
+Node: dcid:medical_Genomics_Laboratory_Department_of_Genetics_UAB_ID
+name: "medical_Genomics_Laboratory_Department_of_Genetics_UAB_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database medical_Genomics_Laboratory_Department_of_Genetics_UAB for a genetic variant."
+
+Node: dcid:laboratory_for_Animal_Genetics_Ghent_University_ID
+name: "laboratory_for_Animal_Genetics_Ghent_University_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database laboratory_for_Animal_Genetics_Ghent_University for a genetic variant."
+
+Node: dcid:geisinger_Autism_and_Developmental_Medicine_Institute_Geisinger_Health_System_ID
+name: "geisinger_Autism_and_Developmental_Medicine_Institute_Geisinger_Health_System_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database geisinger_Autism_and_Developmental_Medicine_Institute_Geisinger_Health_System for a genetic variant."
+
+Node: dcid:clinical_Genetics_Laboratory_Federal_University_of_Health_Sciences_of_Porto_Alegre_ID
+name: "clinical_Genetics_Laboratory_Federal_University_of_Health_Sciences_of_Porto_Alegre_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database clinical_Genetics_Laboratory_Federal_University_of_Health_Sciences_of_Porto_Alegre for a genetic variant."
+
+Node: dcid:genomic_Medicine_Lab_University_of_California_San_Francisco_ID
+name: "genomic_Medicine_Lab_University_of_California_San_Francisco_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database genomic_Medicine_Lab_University_of_California_San_Francisco for a genetic variant."
+
+Node: dcid:klein_lab_Ludwig-Maximilians-University_ID
+name: "klein_lab_Ludwig-Maximilians-University_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database klein_lab_Ludwig-Maximilians-University for a genetic variant."
+
+Node: dcid:foundation_for_Research_in_Genetics_and_Endocrinology_FRIGE's_Institute_of_Human_Genetics_ID
```

Comment:
Could we remove apostrophe?

Reply:
I see shanth had the same comment, so @spiekos  could you remove the ' ?

Reply:
done